arangopipe/arangopipe/arangopipe_storage/arangopipe_api.py

```diff
@@ -46,10 +47,10 @@ class ArangoPipe:
     (4) Register you model with ArangoPipe
     """
 
-    def __init__(self, config: ArangoPipeConfig):
+    def __init__(self, db: StandardDatabase, config: ArangoPipeConfig):
         self.cfg = config.get_cfg()
         self.emlg = None
-        self.db = None
+        self.db = db
         self.mscp = ManagedServiceConnParam()
```

Comment:
Is this still being used anywhere in this class?

Reply:
Yes, methods like `init_graph()` and `lookup_modelperf()` use it.